config/initializers/doorkeeper.rb

```diff
@@ -0,0 +1,25 @@
+# do not auto initialize all the models
+# will make rails runner fail because the models were preloaded
+module Doorkeeper
+  autoload :AccessGrant, 'doorkeeper/orm/active_record/access_grant'
+  autoload :AccessToken, 'doorkeeper/orm/active_record/access_token'
+  autoload :Application, 'doorkeeper/orm/active_record/application'
+
+  module Orm
+    module ActiveRecord
+      def self.initialize_models!
+      end
+    end
+  end
+end
+
+Doorkeeper.configure do
+  orm :active_record
+
+  resource_owner_authenticator do
+    env['warden'].user || redirect_to(login_url)
```

Comment:
we can't "throw(:warden)" because we want to try session auth as well

I think we can get rid of the redirect actually since there will be an auth'd user when we get here.